tests/data/mock_manifests/filepath_submission_test_manifest.csv

```diff
@@ -0,0 +1,3 @@
+Filename,Sample ID,File Format,Component,Genome Build,Genome FASTA,Id,entityId
```

Comment:
Should we also start to put these into: `tests/integration/data`?

Reply:
@BryanFauble I'd rather wait until we do the bigger structure change or do that move separate from this PR as the helper functions will need to be updated as well